carl/learning/calibration.py

```diff
@@ -53,12 +56,21 @@ def _fit_calibrators(self, df0, df1):
                                     range=[(df_min, df_max)],
                                     interpolation="linear")
 
+        elif self.method == "interpolated-isotonic":
+            self.interp_iso_ = InterpolatedIsotonicRegression()
+
+            Xtemp = np.hstack((df0.reshape(-1,),df1.reshape(-1,)))
+            self.interp_iso_.fit(Xtemp.reshape(-1,),
+                np.hstack((np.zeros(df0.size),1.+np.zeros(df1.size))))
+            calibrator0, calibrator1 = clone(self.method), clone(self.method)
```

Comment:
No need for cloning. Just directly `return None, None` after `self.interp_iso_.fit`